code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -133,6 +133,40 @@
 		/obj/item/wrench/medical,
 	)
 
+/obj/item/storage/belt/medical/surgery
+	max_w_class = WEIGHT_CLASS_NORMAL
+	max_combined_w_class = 17
+	use_to_pickup = 1
+	name = "Surgical Belt"
+	desc = "Can hold various surgical tools."
+	storage_slots = 9
+	use_item_overlays = 1
+	can_hold = list(
+		/obj/item/scalpel,
+		/obj/item/hemostat,
+		/obj/item/retractor,
+		/obj/item/circular_saw,
+		/obj/item/bonegel,
+		/obj/item/bonesetter,
+		/obj/item/FixOVein,
+		/obj/item/surgicaldrill,
+		/obj/item/cautery,
+	)
+
+/obj/item/storage/belt/medical/surgery/loaded

```

Comment:
you don't have to explicitly define it, if the properties are the same and it's only `New` that specifies different behavior.

Reply:
Actually, here, I had to modify the storage slots, and weight class, so it could hold all the surgery tools.